Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Shared/Crashes.cs

```diff
@@ -47,11 +47,15 @@ public static void AddNativeType(List<RawType> nativeTypes)
 
         public static void TrackError(Exception exception, IDictionary<string, string> properties = null, params ErrorAttachmentLog[] attachments)
         {
-            if (exception != null)
+            if (exception != null && _reportUnhandledExceptions)
```

Comment:
Not sure about this line - I might be wrong - shouldn't we rather check something like `isEnabled`?

Reply:
This field `_reportUnhandledExceptions` has information about current state Unhandled Exceptions (enabled/disabled) and set in here https://github.com/microsoft/appcenter-sdk-unity/blob/d85217be2e2b4b87aa69941af2dc0d769024dee0/Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Shared/Crashes.cs#L153

Reply:
Yes but by the context of code it looks like not only unhandled exceptions can be here?